CHANGELOG.asciidoc

```diff
@@ -43,6 +43,7 @@ https://github.com/elastic/beats/compare/v5.1.1...master[Check the HEAD diff]
 
 *Filebeat*
 - Fix registry cleanup issue when files falling under ignore_older after restart. {issue}2818[2818]
+- Fix registry migration issue from old states were files were only harvested after second restart.
```

Comment:
will do a second commit. Linking to PR is always tricky before it is created :-)